Dockerfile

```diff
@@ -15,41 +15,22 @@ RUN apt-get update && \
     build-essential libssl-dev libffi-dev python3-dev \
     && rm -rf /var/lib/apt/lists/*
 
-# Clone home-assistant/core
-RUN git clone --depth 1 https://github.com/home-assistant/core.git /core
-
-# Install home-assistant/core dependencies
-RUN pip3 install -r /core/requirements.txt --use-pep517 && \
-    pip3 install -r /core/requirements_test.txt --use-pep517 && \
-    pip3 install -e /core/ --use-pep517
-
-# Clone the Adaptive Lighting repository
-RUN git clone https://github.com/basnijholt/adaptive-lighting.git /app
-
-# Setup symlinks in core
-RUN ln -s /app/custom_components/adaptive_lighting /core/homeassistant/components/adaptive_lighting && \
-    ln -s /app/tests /core/tests/components/adaptive_lighting && \
-    # For test_dependencies.py
-    ln -s /core /app/core
+COPY . /app/
 
 # Install dependencies of components that Adaptive Lighting depends on
-RUN pip3 install $(python3 /app/test_dependencies.py) --use-pep517
+RUN pip3 install -r /app/requirements.txt
 
-WORKDIR /core
+WORKDIR /app
 
-ENTRYPOINT ["python3", \
-    # Enable Python development mode
-    "-X", "dev", \
-    # Run pytest
-    "-m", "pytest", \
+ENTRYPOINT ["pytest", \
```

Comment:
How important is the Python dev mode? Is it necessary? I couldn't get the dev mode and coverage collection to work at the same time, at least one test always failed, and they generally executed noticeably slower. Dev mode without coverage works.

Reply:
Reverted, working again